packages/wonder-blocks-modal/components/modal-dialog.js

```diff
@@ -47,10 +78,21 @@ const styleSheets = {
             flexDirection: "row",
             alignItems: "stretch",
             position: "relative",
-
             borderRadius: 4,
             overflow: "hidden",
         },
+        above: {
+            pointerEvents: "none",
+            position: "absolute",
+            transform: "translate(-50%, -50%)",
```

Comment:
Used `translate` here to center `above` and `below` containers. This way devs could have complete freedom to position their custom containers having that central point as initial reference.